app/views/bookmarks/search/index.json.jbuilder

```diff
@@ -0,0 +1,22 @@
+json.success true
+
+json.results do
+  json.bookmarks do
+    json.name "Bookmarks"
+    json.results do
+      json.array! @bookmarks, partial: "bookmarks/bookmark", as: :bookmark
+    end
+  end
+
+  json.tags do
+    json.name "Tags"
+    json.results do
+      json.array! @tags, partial: "bookmarks/tags/tag", as: :tag
+    end
+  end
+end
+
+json.action do
+  json.url bookmarks_search_index_path(q: params[:q])
+  json.text "See all results"
```

Comment:
`See` feels weird here. `Get`?

Reply:
These JSON files are for semantic UI's little tools, like the search module, so this string gets surfaced like so. I think see fits better with the search pattern then, since clicking it takes you to the full search page.

![transientbug bookmarks 2018-01-17 11-16-43](https://user-images.githubusercontent.com/94542/35059314-f3a2a300-fb77-11e7-838f-4c6d6ee11a20.png)

Reply:
...huh.